hazelcast/src/test/java/com/hazelcast/replicatedmap/ReplicatedMapTest.java

```diff
@@ -1030,6 +1031,18 @@ public void testSizeAfterRemove() throws Exception {
         assertTrue(map.size() == 0);
     }
 
+
+    @Test
+    public void testDestroy() throws Exception {
+        HazelcastInstance instance = createHazelcastInstance();
+        ReplicatedMap<Object, Object> replicatedMap = instance.getReplicatedMap(randomName());
+        replicatedMap.put(1, 1);
+        replicatedMap.destroy();
+        Collection<DistributedObject> objects = instance.getDistributedObjects();
```

Comment:
Add an empty line between 'doing' and 'asserting'. This style is recommended on the wiki to make it easier to see what a test is doing.